web/src/admin/outposts/OutpostForm.ts

```diff
@@ -20,14 +22,76 @@ import {
     OutpostTypeEnum,
     OutpostsApi,
     OutpostsServiceConnectionsAllListRequest,
-    PaginatedLDAPProviderList,
-    PaginatedProxyProviderList,
-    PaginatedRACProviderList,
-    PaginatedRadiusProviderList,
+    Pagination,
     ProvidersApi,
     ServiceConnection,
 } from "@goauthentik/api";
 
+interface ProviderBase {
+    pk: number;
+    name: string;
+    assignedBackchannelApplicationName?: string;
+    assignedApplicationName?: string;
+}
+
+interface ProviderData {
```

Comment:
Could probably replace this with `import { PaginatedResponse } from "@goauthentik/app/elements/table/Table";` which is a generic type